content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -95,10 +95,10 @@ At this point, the command has two additional side effects. The command installs
 
 ## 3. Generate Prisma Client for the Data Proxy
 
-Next, generate Prisma Client for the Data Proxy with the `--data-proxy` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
+Next, generate Prisma Client for the Data Proxy with the `--no-engines` flag. Later, you use the [Data Proxy](/data-platform/classic-projects/data-proxy) to connect to your database over HTTP.
 
 ```terminal
-deno run -A --unstable npm:prisma generate --data-proxy
+deno run -A --unstable npm:prisma generate --no-engines
```

Comment:
We should add a note that this is only possible from 5.2.0 and the alternative flag to run if one is running an earlier version. (This applies to the Cloudflare Workers guide too)